configs/matrix_authorization.yaml

```diff
@@ -18,6 +18,16 @@ jenkins:
       - "GROUP:Job/Read:o3de*reviewers"
       - "GROUP:Overall/Read:o3de*reviewers"
       - "GROUP:View/Read:o3de*reviewers"
+      - "GROUP:Job/Build:o3de*sig-graphics-audio-maintainers"
+      - "GROUP:Job/Cancel:o3de*sig-graphics-audio-maintainers"
+      - "GROUP:Job/Read:o3de*sig-graphics-audio-maintainers"
+      - "GROUP:Overall/Read:o3de*sig-graphics-audio-maintainers"
+      - "GROUP:View/Read:o3de*sig-graphics-audio-maintainers"
+      - "GROUP:Job/Build:o3de*sig-graphics-audio-reviewers"
+      - "GROUP:Job/Cancel:o3de*sig-graphics-audio-reviewers"
```

Comment:
Would it be better to do this or figure out why nested sig groups in reviewers and maintainers are not being surfaced to the overall permission?

Reply:
A shame. Which means this needs to be maintained. Maybe we should just put in all of the sig review and maintainer groups, then?